scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -654,7 +654,7 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
       var hasTypes = false
 
       @tailrec
-      def paramOrType(): Type = token match {
+      def paramOrType()(implicit modsBuf: mutable.Builder[Mod, List[Mod]]): Type = token match {
```

Comment:
Is making it implicit necessary?

Reply:
of course, not 😀 old habits, reducing size of diff.

Reply:
done